src/plugins/cloudinary-analytics/index.js

```diff
@@ -1,10 +1,14 @@
-import { connectCloudinaryAnalytics } from 'cloudinary-video-analytics';
+import videojs from 'video.js';
+import { connectCloudinaryAnalytics } from '../../../../cloudinary-video-analytics';
 import { PLAYER_EVENT } from '../../utils/consts';
 
 class CloudinaryAnalytics {
   constructor(player) {
     this.player = player;
-    this.cloudinaryAnalytics = connectCloudinaryAnalytics(this.player.videoElement);
+    this.shouldUseCustomEvents = videojs.browser.IS_IOS;
+    this.cloudinaryAnalytics = connectCloudinaryAnalytics(this.player.videoElement, {
+      customEvents: this.shouldUseCustomEvents,
+    });
```

Comment:
@jakub-roch [this ticket](https://github.com/videojs/video.js/issues/6679) suggests that you should listen to events on the `player` and not on the `player.videoElement`, did you try that maybe?